### DISCUSSION TITLE
WiFi list redesign

### DISCUSSION THREAD

**Discussion by peteruithoven** [elementary/switchboard-plug-network#380] [Ideas]
## Problem
You need to connect to WiFi networks to configure them or use the out of style seeming "edit connections" UI. 

![Screenshot from 2019-10-30 18 11 57@2x](https://user-images.githubusercontent.com/523210/67881518-ecc01200-fb40-11e9-95f9-2b5324d64e9e.png)

**Describe the solution you'd like**

Redesign with buttons per list item like new VPN UI and current bluetooth UI:
![image](https://user-images.githubusercontent.com/14235191/67739097-45a38380-fa37-11e9-8694-ca298f5bf9d5.png)
![bluetooth](https://user-images.githubusercontent.com/523210/46298641-da6f4f80-c59f-11e8-8ece-ff5d3c4ad4f8.png)

**Additional context**

Relevant: [Editable list HIG](https://github.com/elementary/website/issues/2072)

This issue is split from: https://github.com/elementary/switchboard-plug-network/issues/7 which was partly fixed by https://github.com/elementary/switchboard-plug-network/pull/208.

**Comment by danirabbit** [comment]
We probably need to do something more like the bluetooth UI because we can't adjust settings for networks that haven't been connected to yet. So I think some individual steps to be taken here are:

- [ ] Show settings buttons on any network that has been remembered
- [ ] #120
- [ ] https://github.com/elementary/switchboard-plug-network/issues/250

Once those are completed we should be able to safely:
- [ ] Remove the edit connections button

Some other design changes that could help with consistency:
- [x] Use 32px color icons on the left side of the list and make sure those still reflect signal strength https://github.com/elementary/icons/issues/825
- [x] Badge icons with `security-*` icons
- [x] Add an explicit connect button to the right side of the list item. Perhaps instead of "Connect" it should have some language that indicates the other option will be unselected like "Switch" or "Choose"

**Comment by cassidyjames** [comment]
@danrabbit how about "Nearby" and "Remembered"?
